app/main.py

```diff
@@ -14,4 +14,7 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    if len(phrase) != 0:
+        return phrase.lower().count(letter.lower())
```

Comment:
Looks good, but can be simplified) `if` block is not necessary, the code without it won't produce errors with empty string)

Reply:
yep, you're right)) my fault/